Dockerfile

```diff
@@ -6,7 +6,6 @@ WORKDIR /app
 ADD Pipfile Pipfile.lock ./
 RUN set -e \
  && apt-get update \
- && apt-get install -y --no-install-recommends qrencode \
  && apt-get clean && rm -rf /var/lib/apt/lists/* \
```

Comment:
Since you're removing the only debian package that was being installed, you should also remove the other `apt-get` commands.